vscode_extension/src/types.ts

```diff
@@ -29,7 +29,7 @@ export enum RestartReason {
 }
 
 // Note: Sorbet is either running/in the process or running, or in an error state. There's no benign idle/not running state.
-export const enum ServerStatus {
```

Comment:
This is so we can call `ServerStatus[e.status]`. When you use a `const` enum, that's not possible :(

We could also just send the number as the status, but that looks pretty obscure for 3rd parties trying to do anything with it.

Reply:
Another alternative is to manually maintain a mapping from numbers to names, which I don't love

Reply:
Makes sense - I've run into the same situation in the past.